<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Исходное изображение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Выходное изображение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAddNoise">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>310</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Внести шум</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>260</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Количество шума (%):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldPercentNoise">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>260</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAddImage">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить изображение</string>
    </property>
   </widget>
   <widget class="QLabel" name="originalImage">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>421</width>
      <height>300</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="outputImage">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>421</width>
      <height>300</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="clearField">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить поле</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClearNoise">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>370</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить шум</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonFilter">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>570</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Фильтровать</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderThreshold">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>510</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>540</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Порог</string>
    </property>
   </widget>
   <widget class="QLabel" name="valueSlider">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>510</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSaveImage">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить изображение</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>766</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
